src/components/Buttons.js

```diff
@@ -4,20 +4,200 @@
+        </div>
+        <div className="btn-row">
+          <div className="btn-col">
+            <button
+              type="button"
+              className="cal-btn"
+              onClick={() => setCalObj({ ...calObj, next: parseInt(`${calObj.next}`, 10) || 4 })}
+            >
+              4
+            </button>
+          </div>
+          <div className="btn-col">
+            <button
+              type="button"
+              className="cal-btn"
+              onClick={() => setCalObj({ ...calObj, next: parseInt(`${calObj.next}`, 10) || 5 })}
+            >
+              5
+            </button>
+          </div>
+          <div className="btn-col">
+            <button
+              type="button"
+              className="cal-btn"
+              onClick={() => setCalObj({ ...calObj, next: parseInt(`${calObj.next}`, 10) || 6 })}
+            >
+              6
+            </button>
+          </div>
+          <div className="btn-col">
+            <button
+              type="button"
+              className="cal-btn"
+              onClick={() => setCalObj(calculate({ ...calObj }, '-'))}
+            >
+              -
+            </button>
+          </div>
+        </div>
+        <div className="btn-row">
+          <div className="btn-col">
+            <button
+              type="button"
+              className="cal-btn"
+              onClick={() => setCalObj({ ...calObj, next: parseInt(`${calObj.next}`, 10) || 1 })}
+            >
+              1
+            </button>
+          </div>
+          <div className="btn-col">
+            <button
+              type="button"
+              className="cal-btn"
+              onClick={() => setCalObj({ ...calObj, next: parseInt(`${calObj.next}`, 10) || 3 })}
+            >
+              3
+            </button>
+          </div>
+          <div className="btn-col">
+            <button
+              type="button"
+              className="cal-btn"
+              onClick={() => setCalObj({ ...calObj, next: parseInt(`${calObj.next}`, 10) || 4 })}
+            >
+              4
+            </button>
+          </div>
+          <div className="btn-col">
+            <button
+              type="button"
+              className="cal-btn"
+              onClick={() => setCalObj(calculate({ ...calObj }, '+'))}
+            >
+              +
+            </button>
+          </div>
+        </div>
+        <div className="btn-row">
+          <div className="btn-col">
+            <button
+              type="button"
+              className="cal-btn"
+              onClick={() => setCalObj({ ...calObj, next: parseInt(`${calObj.next}`, 10) || 0 })}
+            >
+              0
+            </button>
+          </div>
+          <div className="btn-col">
+            <button
+              type="button"
+              className="cal-btn"
+              onClick={() => setCalObj(calculate({ ...calObj }, '.'))}
+            >
+              .
+            </button>
```

Comment:
- Whenever I tried to click on this button, It also threw an error. So kindly fix this as well :+1: 
- Error Message 
![image](https://user-images.githubusercontent.com/102699958/228616979-fa5a2778-2a35-420d-b38d-47f5db24aa34.png)